src/scss/attendee/variables.scss

```diff
@@ -23,3 +23,8 @@ $text-chat-height: 40vh;
 $video-chat-height: 160px;
 
 $sparkle-border: rgba(130, 130, 130, 0.3);
+
+$mobile-breakpoint: 768px;
```

Comment:
There is a good opportunity to namespace variables.

I do think putting all constants in a single file named variables is bad design.

First and foremost constants should be divided thematically:
`font.scss`, `mobile.scss` or `breakpoints.scss` etc.

Secondly, names themselves can use suffixes to show variability while sharing prefix.
Example (more in https://github.com/sparkletown/sparkle/blob/staging/src/scss/constants.scss#L121-L126):
```scss

$breakpoint--mobile
$breakpoint--desktop

$font-size--lg
$font-size--md
```